include/oneapi/internal/daal_ze_module_helper.h

```diff
@@ -41,7 +41,21 @@ namespace internal
 {
 namespace interface1
 {
-static const char * zeLoaderName            = "libze_loader.so";
+            #ifdef __linux__
+
+static const char * zeLoaderName = "libze_loader.so";
+static const int libLoadFlags    = RTLD_NOLOAD | RTLD_NOW | RTLD_LOCAL;
+
+            #elif defined(_WIN32) || defined(_WIN64)
+
+static const char * zeLoaderName = "libze_loader.dll";
+static const int libLoadFlags    = 0;
+
+            #else
+
+                #error "Level Zero loader name is not defined"
```

Comment:
Probably need to reformulate, e.g. "Level Zero support is unavailable for this platform"

Reply:
fixed